### PR TITLE
[script.reddit.reader] v0.7.1

### PR DESCRIPTION
**Pull request by gedisony**
### Description
added features and corrected issue brought up by:
Rechi: "this should be resources/fanart.png instead of resources/fanart.jpg"

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [sciprt.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.reddit.reader/addon.xml [hunk 1]
```diff
@@ -2,14 +2,15 @@
 <addon
     id="script.reddit.reader"
     name="Reddit Reader"
-    version="0.6.2"
+    version="0.7.1"
     provider-name="gedisony">
     <requires>
-        <import addon="xbmc.python"                     version="2.25.0"/>
+        <import addon="xbmc.python"                     version="2.20.0"/>
```
Comment (enen92) [inline script.reddit.reader/addon.xml]: please use 2.25.0 (krypton branch)

File: script.reddit.reader/default.py [hunk 2]
```diff
@@ -121,7 +133,9 @@
     nsfw = "nsfw:no+"
 
 def log(message, level=xbmc.LOGNOTICE):
```
Comment (enen92) [inline script.reddit.reader/default.py]: default should be xbmc.LOGDEBUG

File: script.reddit.reader/resources/lib/httpd.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
+#file httpd.py
+#http://forum.kodi.tv/showthread.php?tid=204064
+
+import threading
+import thread
+import sys
+from cgi import parse_qs
+from wsgiref.simple_server import make_server
+if sys.version_info >=  (2, 7):
```
Comment (enen92) [inline script.reddit.reader/resources/lib/httpd.py]: this is not needed anymore, you can use just `import json`

File: script.reddit.reader/resources/lib/httpd.py [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    import json as json
+else:
+    import simplejson as json
+
+
+import xbmc
+
+
+def log(message, level=xbmc.LOGNOTICE):
```
Comment (enen92) [inline script.reddit.reader/resources/lib/httpd.py]: please use DEBUG as default

**Comment by enen92** [comment]
Looks okay, please adjust my comments to merge

**Comment by gedisony** [comment]
Thanks for your work. I've updated the code to match with your recommendation.

**Pull request merged**